src/python/grapl-common/grapl_common/time_utils.py

```diff
@@ -4,18 +4,21 @@
 # we have a bunch of bare ints around the codebase
 # representing timestamps, this makes it slightly easier
 # to reason about
-Millis = NewType("Millis", int)
+MillisSinceEpoch = NewType("MillisSinceEpoch", int)
```

Comment:
I appreciate the clarifying type name here, but it's a bit surprising that we should have a timer that cares about how many milliseconds have elapsed since the epoch.

Reply:
Not timers necessarily; this is a nice way to label our dgraph types that are just "created_at: UInt! @create_time" without hungarian notation'ing them with `_ms` at the end